lib/cc/analyzer/container.rb

```diff
@@ -124,6 +124,15 @@ def container_data(duration: nil, status: nil)
           @stderr_io.string
         )
       end
+
+      def reap_running_container
+        container_id = `docker ps --quiet --filter name=#{@name}`.chomp
```

Comment:
`docker kill` (and all docker commands0 accept a container name as a substitute for the container ID. Therefore, I think the name lookup is unnessary.